src/modules/ens/selectors.ts

```diff
@@ -42,6 +42,11 @@ export const getExternalNamesForConnectedWallet = createSelector(getExternalName
   return Object.values(externalNames).filter(externalName => isEqual(externalName.nftOwnerAddress, address))
 })
 
+export const getExternalNamesForWallet = (wallet: string) => (state: RootState) => {
```

Comment:
Would you mind adding tests for this new selector?

Reply:
Added :D